src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -64,7 +64,8 @@ public async Task FindReferencesAsync(ISymbol symbol, CancellationToken cancella
             await _progress.OnStartedAsync(cancellationToken).ConfigureAwait(false);
             try
             {
-                await using var _ = await _progressTracker.AddSingleItemAsync(cancellationToken).ConfigureAwait(false);
+                var disposable = await _progressTracker.AddSingleItemAsync(cancellationToken).ConfigureAwait(false);
+                await using var _ = disposable.ConfigureAwait(false);
```

Comment:
💡 The separate form is typically for cases where `disposable` needs to be used. For this code, I would typically combine these into a single line (it's a long line, but that's acceptable).

Reply:
I chatted with @CyrusNajmabadi offline about this and he expressed a preference to factor it this way rather than as a single line.